src/redux/categories/catSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+import CatItems from './CatItems';
+
+const initialState = {
+  categories: CatItems,
+};
+
+const catSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: (state, action) => {
+      // Update categories only if payload is 'Under construction'
+      state.categories = action.payload === 'Under construction' ? 'Under construction' : CatItems;
```

Comment:
- [ ] As requested by the previous reviewer you should use state.categories instead of CatItems when the payload is not equal to "Under construction", remember that CatItems is your initial state.